salto-io/salto
# Bump to version 0.3.49

# Salto Version 0.3.49

## New features and bug fixes

_Core_:
- Added environment flag `SALTO_DISABLE_RESOLVE_TYPES_IN_ELEMENT_SOURCE` that allows turning off type resolving in fetch / deploy
- Add `skipValidations` flag to preview API

_Salesforce Adapter_:
- Removed the `valueSetFilter`, which used to set deleted values of Picklist field value sets as "inactive" upon successful deployment

_Jira Adapter_:
- Add support for deploying JSM request types
- Add support for JSM forms (fetch and deploy)
- Add support for fetching JSM asset schemas
- Add support for fetching JSM asset statuses
- Resolve issue that caused HTTP error code 415 on some automation deletions

_Zendesk Adapter_:
- Remove validation for user and organization fields
- Add validation to prevent addition of a dynamic content with no default variant
- Child elements will be omitted when their parent is omitted due to elemId collision
- When the `author_id` in articles is a numerical value, we will no longer generate a fetch warning
- It is possible to add the email username as idField for support_address using `&username`
- Remove empty strings from side conversation group template expressions
- Improve performance of large deployments with no guide related changes
- Deploying order objects can now occur without all of the list items. This can assist deploying between environments that do not have some of the items in the list

_Okta_adapter_:
- Automation and AutomationRule elements will be added

_Netsuite Adapter_:
- Support excluding elements by criteria (e.g `isinactive = true`)
```
netsuite {
  ...
  exclude = {
    ...
    types = [
      ...
      {
         name = ".*"
         criteria = {
            isinactive = true
         }
      },
    ]
  }
}
```
